GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -104,6 +105,7 @@ public GitStatusMonitor(IGitUICommandsSource commandsSource)
             Init(commandsSource);
 
             _getAllChangedFilesOutputParser = new GetAllChangedFilesOutputParser(() => commandsSource.UICommands.GitModule);
+            _isFirstPostRepoChanged = false;
```

Comment:
Can be omitted because it is the default value, isn't it?

Reply:
Yes, it was added for clarity but I will remove it